lib/src/Uncertainty/Model/openturns/DistributionImplementation.hxx

```diff
@@ -1334,6 +1337,79 @@ protected:
     const DistributionImplementation * p_distribution_;
   }; // class ConditionalCDFWrapper
 
+  // Class used to compute entropy
+  class EntropyKernel: public FunctionImplementation
```

Comment:
This should go to the cxx file, or swig will try to wrap it (or at least be private).

Reply:
@jschueller You mean, with all the other methods bindings? Fun...

Reply:
Or at the very least not be public. My concern is swig trying to wrap it.

Reply:
@jschueller I don't understand why SWIG is trying to do something here. The definition is in the protected section of the definition of the class, and is supposed to be hidden to SWIG by a preprocessor conditional directive (#ifndef SWIG / #endif). Do you have an explanation why SWIG still try to do something here? It should never see this code!

Reply:
Sorry, I thought it was in the public section.
SWIG does not parse protected sections, so the ifdefs should be useless here.